tools/xhprof/README.md

```diff
@@ -0,0 +1,17 @@
+Using XHProf with the WordPress core development environment
+============================================================
+
+[XHProf](https://www.php.net/manual/en/book.xhprof.php) is an open source PHP profile.
+
+### Setup with WordPress core development environment
+
+1. git clone https://github.com/WordPress/wordpress-develop.git
+2. Setup the environment using [these](https://github.com/wordPress/wordpress-develop/#development-environment-commands) instructions.
+3. Start environment, using `npm run env:start`
+4. Go to http://localhost:8889 and setup site.
+5. Stop environment, using `npm run env:stop`
+6. Copy docker-compose.override.yml into the root of wordpress-develop directory.
+7. Copy advanced-cache.php into the src/wp-content directory.
+8. Add `define( 'WP_CACHE', true );` to wp-config.php ( to enable profiler ).
```

Comment:
These two steps we could simplify. Maybe just create a completely new PHP file with the profiler code (instead of `advanced-cache.php`) and then change this documentation that they need to `require_once` the file in `wp-config.php`?

Reply:
Fixed